pkg/virtctl/vnc/vnc.go

```diff
@@ -39,7 +40,7 @@ import (
 )
 
 const (
-	LISTEN_TIMEOUT = 60 * time.Second
+	LISTEN_TIMEOUT = 180 * time.Second
```

Comment:
Why is the timeout increased?

Reply:
Ah ok. I think we have to change that. If the proxy option is set, we should probably not timeout.